src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,47 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+    List<Long> eurekaList = new ArrayList<Long>();
```

Comment:
why global variable? The class contains an algorithm, does it need a state? Can be problematic for concurrency etc.